frontend/src/modules/contributor/components/edit/contributor-add.vue

```diff
@@ -0,0 +1,264 @@
+import LfScrollShadow from '@/ui-kit/scrollshadow/ScrollShadow.vue';
+import { ContributorApiService } from '@/modules/contributor/services/contributor.api.service';
+import { useRouter } from 'vue-router';
+import { storeToRefs } from 'pinia';
+import { useLfSegmentsStore } from '@/modules/lf/segments/store';
+import Message from '@/shared/message/message';
+
+const props = defineProps<{
+  modelValue: boolean,
+}>();
+
+const emit = defineEmits<{(e: 'update:modelValue', value: boolean): void }>();
+
+const isModalOpen = computed({
+  get: () => props.modelValue,
+  set: (value) => emit('update:modelValue', value),
+});
+
+const router = useRouter();
+
+const { selectedProjectGroup } = storeToRefs(useLfSegmentsStore());
+
+type ContributorAddIdentity = ContributorIdentity & {
+  image: string,
+  prefix: string,
+  placeholder: string,
+}
+
+const platformList: ContributorAddIdentity[] = Object.entries(CrowdIntegrations.memberIdentities)
+  .map(([key, config]) => ({
+    prefix: config.urlPrefix,
+    placeholder: config.placeholder,
+    image: config.image,
+    platform: key,
+    type: 'username',
+    value: '',
+    verified: true,
+    sourceId: null,
+  }));
+
+interface ContributorAddForm {
+  subproject: string;
+  name: string;
+  email: string[];
+  identities: ContributorAddIdentity[];
+}
+
+const form = reactive<ContributorAddForm>({
+  subproject: '',
+  name: '',
+  email: [''],
+  identities: platformList,
+});
+
+const rules = {
+  subproject: {
+    required,
+  },
+  name: {
+    required,
+  },
+  email: {
+    required: (value: ContributorAddIdentity[]) => value.some((i) => i.length),
+  },
+  identities: {
+    required: (value: ContributorAddIdentity[]) => value.some((i) => i.value.length),
+  },
+};
+
+const $v = useVuelidate(rules, form);
+
+const sending = ref<boolean>(false);
+const createContributor = () => {
+  if ($v.value.$invalid) {
+    return;
+  }
+
+  const data: Partial<Contributor> = {
+    displayName: form.name,
+    identities: [
+      ...form.identities
+        .filter((i) => !!i.value)
+        .map((i) => ({
+          type: 'username',
+          platform: i.platform,
+          value: i.value,
+          verified: true,
+          sourceId: null,
+        })),
+      ...form.email
+        .filter((email) => !!email)
+        .map((email) => ({
+          type: 'email',
+          platform: 'custom',
+          value: email,
+          verified: true,
+          sourceId: null,
+        })),
+    ],
+    username: {},
+  };
+
+  sending.value = true;
+
+  ContributorApiService.create(data, [form.subproject])
+    .then((newMember) => {
+      router.push({
+        name: 'memberView',
+        params: {
+          id: newMember.id,
+        },
+        query: {
+          projectGroup: selectedProjectGroup.value?.id,
+        },
+      });
+      isModalOpen.value = false;
+    })
+    .catch(() => {
+      Message.error('There was an error creating a person');
+    })
```

Comment:
There is a very specific error that backend shows: 400 "Member with the same platform and username already exists in this tenant".

where we shouldn't simply show "there was an error creating a person".

Can we show a better message for when this happens? Suggestion: "We couldn't add a new person to your community, because one of the identities already exist in another profile."
Also if you check the code, we already have backend returning 409 instead of 400 when this happens. So perhaps you can update the backend error code so that frontend can handle the message display a bit better.

And as a fallback message: "There was an error adding a new person to your community."